compiler/rustc_middle/src/ty/context.rs

```diff
@@ -1267,6 +1267,10 @@ impl<'tcx> TyCtxt<'tcx> {
         self.sess.consider_optimizing(cname.as_str(), msg)
     }
 
+    pub fn debugger_visualizers(self) -> &'tcx Vec<rustc_span::DebuggerVisualizerFile> {
+        self.get_debugger_visualizers(())
+    }
```

Comment:
I don't think this function is necessary.

Reply:
Removed.